lib/run.js

```diff
@@ -139,12 +139,15 @@ function run (opts, cb) {
       clearInterval(interval)
       clients.forEach((client) => client.destroy())
       let result = {
+        title: opts.title,
         requests: histAsObj(requests, totalRequests),
         latency: addPercentiles(latencies, histAsObj(latencies)),
         throughput: histAsObj(throughput, totalBytes),
         errors: errors,
         timeouts: timeouts,
         duration: Math.round((Date.now() - startTime) / 1000),
+        start: startTime,
+        finish: Date.now(),
```

Comment:
please do, those output will very likely be consumed by humans, and `Date` do jsonify.